tfx_bsl/beam/benchmarks/run_inference_benchmark.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to use run_inference from command line
+Below is a complete command line for running this script
+for benchmarks
+
+python3 run_inference_benchemark.py \
+PATH_TO_MODEL \
+PATH_TO_DATA \
+--output gs://YOUR_BUCKET/results/output \
+--project YOUR_PROJECT \
+--runner DataflowRunner \
+--temp_location gs://YOUR_BUCKET/temp \
+--job_name run-inference-metrics \
+--region us-central1
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import apache_beam as beam
+from tfx_bsl.tfxio import raw_tf_record
+from tfx_bsl.beam import run_inference
+from tfx_bsl.public.proto import model_spec_pb2
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.options.pipeline_options import SetupOptions
+
+
+def run(argv=None, save_main_session=True):
+    """Main entry point; defines and runs the user_score pipeline."""
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        'model_path',
+        type=str,
+        help='The path to input model')
+    parser.add_argument(
+        'input',
+        type=str,
+        help='Path to the data file(s) containing data.')
+    parser.add_argument(
+        '--output',
```

Comment:
Should it be 'output'?

Reply:
The script will still run without output, which is why I don't think it should be a positional argument. But I made it required as it might be nice to know where the output is